generator/generator_md.py

```diff
@@ -0,0 +1,96 @@
+import sys
```

Comment:
Your pull request: GENERATOR-1022 Added generator_py.md


Your commit: GENERATOR-1022 Added generator_md.py

Your commit: GENERATOR-1022 Added other generator files

Your commit: GENERATOR-1022 Added testgenerator.py

Your commit: GENERATOR-1022 Refactored generator_md.py
Action of your comment must be from ['Added', 'Deleted']
Your commit: GENERATOR-1022Added testgenerator.py
Number of your group must be from ['1021', '1022']
Action of your comment must be from ['Added', 'Deleted']